### PR TITLE
Nakedi api review 20160202

### PR DESCRIPTION
**Pull request by jkliff**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/zalando/nakadi/pull/82" target="_blank">ReviewNinja

File: api/nakadi-event-bus-api.yaml [hunk 1]
```diff
@@ -91,6 +90,9 @@ info:
     * Validation and Enrichment Strategies available in Nakadi are out of this document an can be consulted
     in an external document (TBD link to document).
 
+    * Explicit control of (in case of Kafka as underying messaging infrastructure) topic's creation
```
Comment (rcillo) [inline api/nakadi-event-bus-api.yaml]: underyin -> underlying

File: api/nakadi-event-bus-api.yaml [hunk 2]
```diff
@@ -220,6 +225,10 @@ paths:
           description: Client is not authenticated
           schema:
             $ref: '#/definitions/Problem'
+        '409':
```
Comment (rcillo) [inline api/nakadi-event-bus-api.yaml]: Just to be clear, this is for the duplicated event type name scenario, right?

File: api/nakadi-event-bus-api.yaml [hunk 3]
```diff
@@ -920,8 +932,8 @@ definitions:
           Name of this EventType. Encodes the owner/responsible for this EventType.
 
           The name for the EventType SHOULD follow the pattern, but is not enforced
-          'organization:team:owning_application:event-type', for example
```
Comment (rcillo) [inline api/nakadi-event-bus-api.yaml]: Is there a size limit? Under 256 chars?

File: api/nakadi-event-bus-api.yaml [hunk 4]
```diff
@@ -953,29 +965,27 @@ definitions:
...
       effective_schema:
         type: string
         #read-only: true
         description: |
           The effective schema of this `EventType`. The predefined schema validator will use the value of
           this field as base for the validation.
-          The generation of this value happens on creation of the `EventType` and is influenced by the
-          `EventType.type`.
+
+          The createion of EventTypes of different categories implies a distinct general structure for the
```
Comment (rcillo) [inline api/nakadi-event-bus-api.yaml]: /createion/creation/

File: api/nakadi-event-bus-api.yaml [hunk 5]
```diff
@@ -953,29 +965,27 @@ definitions:
...
+          payload of events of this each type. This ultimate format the payload must adhere to is reflected
```
Comment (rcillo) [inline api/nakadi-event-bus-api.yaml]: --this--

Comment (rcillo) [inline api/nakadi-event-bus-api.yaml]: I don't know if I correctly understood the sentence "This ultimate format the payload must adhere to is reflected the `effective_schema`". Could you please rephrase it?

File: api/nakadi-event-bus-api.yaml (hunk 3 above)
Comment (jkliff) [inline api/nakadi-event-bus-api.yaml]: No explicit restrictions exist at this time.

**Comment by rcillo** [comment]
Looks good :+1:  :shipit:

**Pull request merged**